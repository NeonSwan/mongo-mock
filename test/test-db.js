var mongodb = require("../");

var server = new mongodb.Server("host", 27017),
    db = new mongodb.Db("database", server);

console.log("db:");
console.log("  host:27017/database -> " + db);
console.log("");
