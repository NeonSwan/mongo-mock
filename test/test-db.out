db:
  host:27017/database -> host:27017/database

